tools/lint.js

```diff
@@ -128,7 +134,10 @@ async function clippy() {
     stdout: "inherit",
     stderr: "inherit",
   });
-  if (!success) {
+  cargoCmd.spawn();
```

Comment:
I don't believe `cmd.spawn()` is needed in this case, as you're not doing anything fancy with the output.

Reply:
@crowlKats @iuioiua TIL, thanks